### PR TITLE
Wallet gas selector redesign

### LINKED ISSUE #2536: new send transaction modal
design here https://www.figma.com/file/XK7vrp3pZdQ5gqQVJ7Pqgz/EIP-1559-Presentation?node-id=80%3A65

note that we can implement most of this, without the gas tip limit which should be done in a different task

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #2536

Probably also fixes the gas validation error on ENS

### CODE REVIEW

File: ui/app/AppMain.qml [hunk 1]
```diff
@@ -284,6 +284,7 @@ RowLayout {
     // Add SendModal here as it is used by the Wallet as well as the Browser
     Loader {
         id: sendModal
+        active: fale
```
Comment (richard-ramos) [inline ui/app/AppMain.qml:287]:
```suggestion
        active: false
```

**Review by iurimatias — APPROVED** [review]

**Comment by anastasiyaig** [comment]
@jrainville can you please rebase, because the PR is failing on pending transactions and it was fixed recently 

`ERR 2021-05-31 15:08:41+03:00 rpc response error                         topics="rpc" tid=1815751 file=core.nim:24 methodName=wallet_watchTransaction payload="[\"0x753f284f4463dab73d863b747b335aa645efcbbb69420656cf98bfe888f724e0\"]" result="{\"jsonrpc\":\"2.0\",\"id\":0,\"error\":{\"code\":-32000,\"message\":\"context deadline exceeded\"}}"
Traceback (most recent call last)
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/src/nim_status_client.nim(220) nim_status_client
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/src/nim_status_client.nim(217) mainProc
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/vendor/nimqml/src/nimqml/private/qapplication.nim(8) exec
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/vendor/nimqml/src/nimqml/private/qobject.nim(49) qobjectCallback
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/vendor/nimqml/src/nimqml/private/qobject.nim(34) onSlotCalled
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/vendor/nimbus-build-system/vendor/Nim/lib/core/macros.nim(598) onSlotCalled
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/src/app/wallet/view.nim(546) transactionWatchResultReceived
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
`

`ERR 2021-05-31 15:12:44+03:00 TODO: if user is logged in, logout         topics="main" tid=1859967 file=nim_status_client.nim:160
Traceback (most recent call last)
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/src/nim_status_client.nim(220) nim_status_client
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/src/nim_status_client.nim(164) mainProc
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/src/status/signals/core.nim(26) delete
/Users/jenkins/workspace/s-desktop_branches_macos_PR-2596/vendor/nimqml/src/nimqml/private/qobject.nim(64) delete
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
`

**Comment by jrainville** [comment]
@anastasiyaig rebased

**Comment by anastasiyaig** [comment]
The redesigned screen of gas selection looks great. Please note that this PR does not fix the problem of sending transactions in chat, so I can't test it there. 
Also, logged https://github.com/status-im/status-desktop/issues/2623

**Comment by iurimatias** [comment]
I'm getting a lot of crashes in this branch with `Error: unhandled exception: key not found: transactionHash [KeyError]`, will hold on merging this for now

**Pull request merged**